os/android/iahwc2.cpp

```diff
@@ -725,7 +725,6 @@ HWC2::Error IAHWC2::HwcDisplay::SetClientTarget(buffer_handle_t target,
 
 HWC2::Error IAHWC2::HwcDisplay::SetColorMode(int32_t mode) {
   supported(__func__);
-  color_mode_ = mode;
```

Comment:
we should add proper implementation for this function, perhaps add some log here to indicate that ?

Reply:
@kalyankondapally Do you mean just adding a fixme / todo note? The fix should probably go in a different pr.

Reply:
@stripes416 Ya, just a Fixme/TODO note.